### DISCUSSION TITLE
Semantic Image Search Support?

### DISCUSSION THREAD

**Discussion by KMJ-007** [meilisearch/product#786] [Feedback & Feature Proposal]
Hi there!

I’m curious if there are plans to support semantic image search in MeiliSearch, specifically text-to-image and image-to-image functionalities. This feature could enhance search capabilities and improve user experience significantly.

Looking forward to hearing your thoughts!

Thanks!

**Comment by macraig** [comment]
Hi @KMJ-007, our AI-powered search feature are Experimental for now. We're working on stabilizing the feature and hope to release it in early 2025. Once the feature is stabilized, we're planning on adding more features like the one you mentioned. 

To follow along with our current AI development, you can post your feedback and feature requests in our dedicated [GitHub discussion](https://github.com/orgs/meilisearch/discussions/677).

**Reply by dureuill** [reply]
In addition to María's answer, it is already possible to send vector data corresponding to an image an indexing and at search time. So it is possible to do semantic image search in Meilisearch, although Meilisearch doesn't natively supports it and so won't help passing and retrieving images.
